### ISSUE TITLE
Ref typings not defined in Typescript

### ISSUE THREAD

**Issue by ramonvermeulen** [BugiDev/react-native-calendar-strip#168]
When making use of createRef with typings in typescript, the methods are not recognized.

```
const calendarStripRef = createRef<CalendarStrip>();

...
...

<StyledCalendarStrip
          {...props}
          ref={calendarStripRef}
/>
```

![image](https://user-images.githubusercontent.com/18006244/74234539-4baf2e80-4ccd-11ea-9d2c-a7d81239516c.png)

I'll make a pullrequest to fix this later.

**Comment by ramonvermeulen** [comment]
Update:

Pull request: https://github.com/BugiDev/react-native-calendar-strip/pull/169

**Comment by ramonvermeulen** [comment]
Pull request is merged. Issue can be closed.

**Comment by AjitZK** [comment]
My IDE, Webstorm, still can't recognize the typings even though I have checked that my version includes the fix from #169 

I'm using `useRef` and not `createRef` if that matters.
